dace/subsets.py

```diff
@@ -176,6 +176,30 @@ def size(self, for_codegen=False):
                 for (iMin, iMax, step), ts in zip(self.ranges, self.tile_sizes)
             ]
 
+    def size_exact(self, for_codegen=False):
+        """ Returns the number of elements in each dimension. """
+        if for_codegen == True:
+            int_ceil = sp.Function('int_ceil')
+            return [
+                ts * int_ceil(
+                    ((iMax.expr if isinstance(iMax, symbolic.SymExpr) else
+                      iMax) + 1 - (iMin.expr if isinstance(
+                          iMin, symbolic.SymExpr) else iMin)),
+                    (step.expr
+                     if isinstance(step, symbolic.SymExpr) else step))
+                for (iMin, iMax, step), ts in zip(self.ranges, self.tile_sizes)
+            ]
```

Comment:
@alexnick83 If no one is calling this mode, do we need it?

Reply:
Also, this is why we have `symcpp` in the codegen, it makes the conversion there.

Reply:
I actually don't know why this mode exists. I just copied it as it was in the size method. Is it used there anywhere?

Reply:
Not sure. We can remove it then

Reply:
I removed the path. We can just re-add it if we find a use-case for it.